### DISCUSSION TITLE
Use of undeclared identifier 'FIRConsentTypeAdPersonalization'

### DISCUSSION THREAD

**Discussion by andreydos** [invertase/react-native-firebase#7713] [Q&A]
I had updated RNFB to the most recent version and got the error: ios/RNFBAnalytics/RNFBAnalyticsModule.m:194:7 Use of undeclared identifier 'FIRConsentTypeAdPersonalization'
    "@react-native-firebase/analytics": "^19.1.1",
    "@react-native-firebase/app": "^19.1.1",
    "@react-native-firebase/crashlytics": "^19.1.1",
    "@react-native-firebase/installations": "^19.1.1",
    "@react-native-firebase/messaging": "^19.1.1",

On version 17.5 my project works perfectly. I can't find anything about this error on internet

**Comment by sirsaula** [comment]
Did you resolve this? i run into it also

**Comment by iamabdulhaseeb** [comment]
Make sure you have updated to the latest version of the Firebase SDK. The FIRConsentTypeAdPersonalization identifier might not be available in older versions. I resolved this issue by installing the latest version of @react-native-firebase/app, and now everything is working smoothly!
